src/input-moment.js

```diff
@@ -36,23 +41,27 @@ export default class InputMoment extends Component {
       minStep,
       hourStep,
       onSave,
+      tab,
```

Comment:
Exclude `tab` value from rest of props passed to div as it would be an unknown property.